### PR TITLE
Application codes and README doc. for Amazon Web Services

### PR DESCRIPTION
**Pull request by youngeunpark**
We succeeded in passing all AWS Test cases to get certification, and now working on publicity of codes based on TizenRT for AWS team to verify.

### CODE REVIEW

File: apps/examples/aws/samples/TizenRT/aws_sample_main.c [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    args.argc = argc;
+    args.argv = argv;
+
+
+    /* Initialize the attribute variable */
+    if ((ret = pthread_attr_init(&attr)) != 0)
+        printf("%s: pthread_attr_init failed, status=%d\n", __func__, ret);
+
+    /* set a stacksize */
+    if ((ret = pthread_attr_setstacksize(&attr, CONFIG_EXAMPLES_AWS_STACKSIZE)) != 0)
```
Comment (sunghan-chang) [inline apps/examples/aws/samples/TizenRT/aws_sample_main.c]: I can not find CONFIG_EXAMPLES_AWS_STACKSIZE on Kconfig.

**Comment by sunghan-chang** [comment]
If you want to add some files except extern folder, you should follow Tizen coding style and add Samsung license and copyright.

**Comment by youngeunpark** [comment]
Hello Sunghan,
I'd like to add more folks on this pull-request, thus they also receive your review comments. 
Would you guide me how to do that ?
Regards,
YE

**Comment by sunghan-chang** [comment]
@youngeunpark You can use assignees at right side of this P.R.

**Comment by sunghan-chang** [comment]
I think you can not assign due to permission. If you write user names, I can assign this them.

**Comment by youngeunpark** [comment]
Would you assign ShinkyuPark ?

**Comment by sunghan-chang** [comment]
@youngeunpark I am sorry, I can not assign. He is not shown at assign tab. I am checking how to do it.

**Comment by ShinkyuPark** [comment]
I moved the AWS IoT SDK code from apps folder to external folder.

**Comment by sunghan-chang** [comment]
@ShinkyuPark Do you need to leave a history moving codes from example to external? And there are many unnecessary commit. Could you please merge your same commits like "Update README file for SIDK S5JT200 board"?

**Comment by youngeunpark** [comment]
We will combine those commits into 1 or 2

**Pull request closed without merging**